src/components/problem/ProblemPage.tsx

```diff
@@ -282,7 +283,7 @@ const ProblemPage = () => {
 
         <div className={"container p-0 pt-3 pb-3 " + state.appState.theme.bg}>
           <div className={"h-100 text-center pb-3 " + state.appState.theme.bg}>
-            <table
+            {problemList?.problems?.length ? <table
```

Comment:
It is not the correct approach, as the problem list can be empty because of an error. You can handle it just like it is handled on the contest page.

Reply:
changed it. used state.problemList.loading